src/main/resources/application.yml

```diff
@@ -0,0 +1,80 @@
+camel:
+  health:
+    enabled: true
+    routes-enabled: true
+  component:
+    rabbitmq:
+      hostname: localhost
+      port-number: 5672
+      username: guest
+      password: guest
+  springboot:
+    main-run-controller: true
+
+spring:
+  application:
+    name: iroha-balancer
+  jmx:
+    enabled: false
+  main:
+    allow-bean-definition-overriding: true
+  task:
+    execution:
+      thread-name-prefix: itps-task-
+      pool:
+        core-size: 2
+        max-size: 50
+        queue-capacity: 10000
+    scheduling:
+      thread-name-prefix: iroha-balancer-scheduling-
+      pool:
+        size: 2
+  rabbitmq:
```

Comment:
Can you put it in README too?

Reply:
this is just for rabbit diagnostic
in readme will be camel paramters